jscripting/mc.d.ts

```diff
@@ -1,4 +1,4 @@
-
```

Comment:
This whole file is autogenerated and should not be modified by hand which it looks like you did.

Reply:
@olee I was under the impression that the auto generation was never fully implemented?

Reply:
Of course it was!
I mean I took all that time to write that generater which used Javadoc for creating definition files.

Reply:
The comments suggested otherwise.

https://github.com/ForgeEssentials/ForgeEssentials/blob/develop/build.gradle#L228

Reply:
@olee

Reply:
That's just the gradle task - the generation needs to be run from IDE I think - there was some run configuration / Java main file for that.

Reply:
https://github.com/ForgeEssentials/ForgeEssentials/blob/develop/src/main/java/com/forgeessentials/jscripting/TsdGenerator.java

Reply:
ok